packages/ejson/package.js

```diff
@@ -5,6 +5,7 @@ Package.describe({
 
 Package.onUse(function onUse(api) {
   api.use(['ecmascript', 'base64']);
+  api.addAssets('ejson.d.ts', ['client', 'server']);
```

Comment:
1. Do we have to add it as an asset?
2. Do we have to add it as an asset **on the client**?

@piotrpospiech, @zodern.

Reply:
1. Yes, without adding it to the assets, it won't be added to the package-types
2. I tested with just a server. It worked, so I will change it.